### PR TITLE
atool, dnf, wl-copy, wl-paste: add Turkish translation

### PR DESCRIPTION
**Pull request by ersen0**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/wl-paste.md [hunk 1]
```diff
@@ -8,18 +8,10 @@
 
 `wl-paste`
 
-- Paste the contents of the clipboard and then clear it:
```
[the remark this answers is not in the document]
Comment (ersen0) [reply pages/linux/wl-paste.md:11]: wl-paste: unrecognized option '--paste-once'

`--paste-once` is for `wl-copy`. With `wl-copy --paste`, you copy content to the clipboard and after one paste, it disappears.

File: pages.tr/linux/atool.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`atool --list {{path/to/archive.zip}}`
+
+- Bir tar.gz arşivini yeni bir alt dizine (veya yalnızca bir dosya içeriyorsa geçerli dizine) çıkart:
+
+`atool --extract {{path/to/archive.tar.gz}}`
+
+- İki dosyaya sahip yeni bir 7zip arşivi oluştur:
+
+`atool --add {{path/to/archive.7z}} {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages.tr/linux/atool.md:16]: Please translate the placeholders (like `path/to/file`) for get this easier for non-English speakers.

File: pages.tr/linux/dnf.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo dnf upgrade`
+
+- Anahtar kelimeler kullanarak paket ara:
+
+`dnf search {{keywords}}`
+
+- Bir paketin aytıntılarını göster:
+
+`dnf info {{package}}`
```
Comment (navarroaxel) [inline pages.tr/linux/dnf.md:16]: Please translate the placeholders for get this easier for non-English speakers.

File: pages.tr/linux/atool.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# atool
+
+> Çeşitli biçimlerdeki arşivleri yönetin.
+> Daha fazla bilgi için: <https://www.nongnu.org/atool/>.
```
Comment (navarroaxel) [inline pages.tr/linux/atool.md:4]: We use this as standard translation
```suggestion
> Daha fazla bilgi: <https://www.nongnu.org/atool/>.
```

File: pages.tr/linux/dnf.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# dnf
+
+> RHEL, Fedora ve CentOS için paket yönetim aracı (yum'um yerini alır).
+> Daha fazla bilgi için: <https://dnf.readthedocs.io>.
```
Comment (navarroaxel) [inline pages.tr/linux/dnf.md:4]:
```suggestion
> Daha fazla bilgi: <https://dnf.readthedocs.io>.
```

File: pages.tr/linux/wl-copy.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# wl-copy
+
+> Wayland için pano değiştirme aracı.
+> Ayrıca bakın: `wl-paste`.
+> Daha fazla bilgi için: <https://github.com/bugaevc/wl-clipboard>.
```
Comment (navarroaxel) [inline pages.tr/linux/wl-copy.md:5]:
```suggestion
> Daha fazla bilgi: <https://github.com/bugaevc/wl-clipboard>.
```

File: pages.tr/linux/wl-paste.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# wl-paste
+
+> Wayland için panoda saklanan verilere erişim aracı.
+> Ayrıca bakın: `wl-copy`.
+> Daha fazla bilgi için: <https://github.com/bugaevc/wl-clipboard>.
```
Comment (navarroaxel) [inline pages.tr/linux/wl-paste.md:5]:
```suggestion
> Daha fazla bilgi: <https://github.com/bugaevc/wl-clipboard>.
```

**Review by navarroaxel — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! 🚀

**Pull request merged**